pkg/controller/volume/scheduling/scheduler_binder.go

```diff
@@ -78,16 +96,16 @@ type InTreeToCSITranslator interface {
 // 2. Once all the assume operations are done in d), the scheduler processes the next Pod in the scheduler queue
 //    while the actual binding operation occurs in the background.
 type SchedulerVolumeBinder interface {
-	// FindPodVolumes checks if all of a Pod's PVCs can be satisfied by the node.
+	// FindPodVolumes checks if all of a Pod's PVC can be satisfied by the node.
```

Comment:
PVCs

Reply:
Oh, bummer. Sorry, fixed.

FWIW, my motivation for the change to "volumes" was that this is the interface documentation, not the documentation for the current implementation of that interface. As far as the caller is concerned, it shouldn't matter whether the implementation checks PVCs or also something else. With this change of booleans with fixed meanings to a slice of strings the caller really doesn't need to care anymore, so it seemed like a good time to replace it.

But I'm also fine with documentation the full behavior here.